design-tokens/build.js

```diff
@@ -30,14 +30,16 @@ const BASE_TOKENS_NAME = `${NAME_PREFIX}base`;
 const CUSTOM_TRANSFORM_GROUP = 'sonar-design-tokens';
 const CUSTOM_FILTER_NO_COLOR = 'sonar-no-color';
 const CUSTOM_FILTER_THEMED_TOKENS = 'sonar-themed-tokens';
-const themeDefinitions = JSON.parse(fs.readFileSync(`${DESIGN_TOKENS_PATH}/$themes.json`, 'utf-8'));
-const licenceHeader = fs.readFileSync(`config/license/LICENSE-HEADER.txt`, 'utf-8');
+const LICENCE_HEADER = fs.readFileSync(`config/license/LICENSE-HEADER.txt`, 'utf-8');
+
+const TOKEN_GROUPS = JSON.parse(fs.readFileSync(`${DESIGN_TOKENS_PATH}/$themes.json`, 'utf-8'));
+const THEMED_TOKEN_GROUPS = TOKEN_GROUPS.filter(({ group }) => group === 'Themes');
```

Comment:
NP: I prefer to use Capitalized constants for true hard-coded values. Here we're reading external data, so I think it makes sense to have "normal" constants. Corollary: don't use the `licenseHeader` directly, but pass it as an argument

Reply:
Ok it make sense